dbr/django_business_rules/business_rule.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import unicode_literals
+
+import json
+
+from business_rules import engine, utils
+
+from django_business_rules.models import BusinessRuleModel
+
+
+class BusinessRule(object):
```

Comment:
This class heavily depends on `BusinessRuleModel` and it has a lot of static methods.
I would think about implementing this class as child of `BusinessRuleModel` (and partially `BusinessRuleManager`).